eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -210,24 +171,9 @@ stages:
                   deploy:
                     steps:
                       - checkout: self
-                      - template: /eng/pipelines/templates/steps/stage-artifacts.yml
-                        parameters:
-                          SourceFolder: ${{parameters.ArtifactName}}-signed
-                          TargetFolder: ${{artifact.safeName}}/packages
-                          FileFilter: "${{artifact.name}}.[0-9]*.[0-9]*.[0-9]*"
-                          Exclude: "*.symbols.nupkg"
-                      - template: /eng/pipelines/templates/steps/stage-artifacts.yml
-                        parameters:
-                          SourceFolder: Docs.${{artifact.name}}
-                          TargetFolder: ${{artifact.safeName}}/Docs.${{artifact.name}}
-                          FileFilter: '*'
-                      - pwsh: |
-                          Get-ChildItem -Recurse $(Pipeline.Workspace)/${{artifact.safeName}}
-                        workingDirectory: $(Pipeline.Workspace)
-                        displayName: Output Visible Artifacts
                       - template: /eng/common/pipelines/templates/steps/publish-blobs.yml
                         parameters:
-                          FolderForUpload: '$(Pipeline.Workspace)/${{artifact.safeName}}'
+                          FolderForUpload: '$(Pipeline.Workspace)/${{parameters.ArtifactName}}/${{artifact.name}}'
                           BlobSASKey: '$(azure-sdk-docs-prod-sas)'
                           BlobName: '$(azure-sdk-docs-prod-blob-name)'
                           TargetLanguage: 'dotnet'
```

Comment:
Need to update ArtifactLocation here. That was one of @sima-zhu's recent changes. We also need to figure out a way to have that only contain the nupkg and not the symbols package otherwise the release tag step will not work.

Reply:
Please work with @sima-zhu to make sure the master to release-tag stuff still works after your change.

Reply:
I copied the `.nupkg` files to another directory just before calling `RetrieveReleaseTag` in the file.